spaceship/cmd/cmd.go

```diff
@@ -23,10 +23,15 @@ var defaultFlags = []*plugin.Flag{
 		Type:      plugin.FlagTypeString,
 	},
 	{
-		Name:  flagPassword,
+		Name:  flagUserPassword,
 		Usage: "ssh user password",
 		Type:  plugin.FlagTypeString,
 	},
+	{
+		Name:  flagPassword,
```

Comment:
Does this really need to be a flag? I'm failing to understand the rationale of this.

Reply:
hm, maybe make more sense we always ask for the password if the user has not set